packages/host-ctr/host-ctr.spec

```
@@ -29,20 +34,21 @@ install -d %{buildroot}%{_cross_bindir}
 install -p -m 0755 host-ctr %{buildroot}%{_cross_bindir}
 
 install -d %{buildroot}%{_cross_unitdir}
-install -p -m 0644 %{S:10} %{buildroot}%{_cross_unitdir}
+install -p -m 0644 %{S:10} "%{S:100}" %{buildroot}%{_cross_unitdir}
 
 install -d %{buildroot}%{_cross_tmpfilesdir}
 install -p -m 0644 %{S:11} %{buildroot}%{_cross_tmpfilesdir}/host-containerd.conf
 
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/host-containerd
 install -p -m 0644 %{S:12} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/host-containerd/config.toml
 
-%cross_scan_attribution go-vendor vendor
+%cross_scan_attribution --clarify %{S:1000} go-vendor vendor
 
 %files
 %{_cross_attribution_vendor_dir}
 %{_cross_bindir}/host-ctr
 %{_cross_unitdir}/host-containerd.service
+"%{_cross_unitdir}/etc-host\x2dcontainers.mount"
```

nit: since quoting these entries is so unusual:
```suggestion
%{_cross_unitdir}/*.mount
```